GOSSIP.md

```diff
@@ -130,32 +130,41 @@ nodes to join the cluster even if the trusted nodes inbound limit has been
 reached. In addition, this prevent a node, in particular a trusted node, to
 have so much inbound connections that it cannot reach its number of outbound
 connections; a guaranteed number of outbound connection is crucial for proper
-propagation of new blocks.
+propagation of new blocks. For all nodes to reach there established number of
+outbound connections, the soft maximum limit for inbound connections must be
+smaller than the maximum number of nodes minus the wanted number of outgoing
```

Comment:
The maximum number of open connections instead of »nodes«, I assume?

Reply:
@knarz I actually mean `total number of nodes` in the cluster. If there are fewer nodes than the maximum number of outbound connection, the trusted nodes will have all the other nodes in the cluster as inbound connections and as such will never reach there the maximum number of outbound connections.